server/src/main/java/io/druid/curator/CuratorModule.java

```diff
@@ -23,48 +23,61 @@
 import com.google.inject.Module;
 import com.google.inject.Provides;
 
-import io.druid.guice.JsonConfigProvider;
-import io.druid.guice.LazySingleton;
-import io.druid.java.util.common.lifecycle.Lifecycle;
-import io.druid.java.util.common.logger.Logger;
-
-import org.apache.curator.framework.api.ACLProvider;
+import org.apache.curator.ensemble.EnsembleProvider;
+import org.apache.curator.ensemble.exhibitor.DefaultExhibitorRestClient;
+import org.apache.curator.ensemble.exhibitor.ExhibitorEnsembleProvider;
+import org.apache.curator.ensemble.exhibitor.Exhibitors;
+import org.apache.curator.ensemble.fixed.FixedEnsembleProvider;
 import org.apache.curator.framework.CuratorFramework;
 import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.framework.api.ACLProvider;
 import org.apache.curator.framework.imps.DefaultACLProvider;
 import org.apache.curator.retry.BoundedExponentialBackoffRetry;
+import org.apache.zookeeper.ZooDefs;
+import org.apache.zookeeper.data.ACL;
 
 import java.io.IOException;
-
 import java.util.List;
 
-import org.apache.zookeeper.ZooDefs;
-import org.apache.zookeeper.data.ACL;
+import io.druid.guice.JsonConfigProvider;
+import io.druid.guice.LazySingleton;
+import io.druid.java.util.common.lifecycle.Lifecycle;
+import io.druid.java.util.common.logger.Logger;
 
 /**
  */
 public class CuratorModule implements Module
 {
+  static final String CURATOR_CONFIG_PREFIX = "druid.zk.service";
+
+  static final String EXHIBITOR_CONFIG_PREFIX = "druid.exhibitor.service";
+
+  private static final int baseSleepTimeMs = 1000;
```

Comment:
general convention is all caps for static. so something like `DEFAULT_BASE_SLEEP_TIME_MS`